### PR TITLE
CFD-329 : formatting bug on vocabulary lists

### PR DESCRIPTION
**Pull request by YuriGizra**
Fix ampersand after sanitization function.

### CODE REVIEW

**Comment by YuriGizra** [comment]
@nader77, @HelenaEksler,  ready for review

**Comment by HelenaEksler** [comment]
@YuriGizra Add screenshot, please.

**Comment by YuriGizra** [comment]
@HelenaEksler 
Screenshot:
selection_132

**Comment by YuriGizra** [comment]
@MPParsley, ready for review. I can't assign, because haven't permissions yet.

File: capacity4more/modules/c4m/vocab/c4m_vocab_widget/c4m_vocab_widget.module [hunk 1]
```diff
@@ -167,7 +167,8 @@ function c4m_vocab_widget_prepare_taxonomy_tree($terms) {
     // If the term is array, this vocab is of type "node".
     if(is_array($term)) {
       foreach($term as $id => $label) {
-        $options[$id]['label'] = check_plain($label);
+        // Decode ampersand back to & after sanitization.
+        $options[$id]['label'] = str_replace("&amp;", "&", check_plain($label));
```
Comment (zero2one) [inline capacity4more/modules/c4m/vocab/c4m_vocab_widget/c4m_vocab_widget.module]: @YuriGizra 
This is not the correct solution: names with accents (Bélizè, Français, …) will still be broken…

Could it be that the labels are already run trough check_plain as the string is shown as `&amp;amp`in the list?

**Comment by nader77** [comment]
Closed in favor of #658

**Pull request closed without merging**